arithmetization/src/main/java/net/consensys/linea/zktracer/module/wcp/Wcp.java

```diff
@@ -119,7 +119,7 @@ public boolean callLT(final Bytes arg1, final Bytes arg2) {
     return this.callLT(Bytes32.leftPad(arg1), Bytes32.leftPad(arg2));
   }
 
-  public boolean callLT(final int arg1, final int arg2) {
+  public boolean callLT(final long arg1, final long arg2) {
```

Comment:
There are places where we want to `callLt` with stack arguments (e.g. in the OOB module), should we not allow for `BigIntegers` here ?

Reply:
we have sereveral `callLT`, for long, Nytes, Bytes32, if needed fot a BigInt, we just add it